shopfloor_reception_measuring_device/services/reception.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo.addons.base_rest.components.service import to_int
+from odoo.addons.component.core import Component
+
+
+class Reception(Component):
+    _inherit = "shopfloor.reception"
+
+    def _get_measuring_device_domain(self):
+        return []
+
+    def set_packaging_dimension__measuring_device_assign(
+        self, picking_id, selected_line_id, packaging_id
+    ):
+        picking = self.env["stock.picking"].sudo().browse(picking_id)
+        selected_line = self.env["stock.move.line"].sudo().browse(selected_line_id)
+        packaging = self.env["product.packaging"].sudo().browse(packaging_id)
+        device_domain = self._get_measuring_device_domain()
+        device = self.env["measuring.device"].search(device_domain)
+        if not packaging:
```

Comment:
IMO this checks should be done right after each browse above

Reply:
IMO here we miss completely the usage of the device to collect measures :thinking: